athena_cli.py

```diff
@@ -42,58 +87,37 @@ def execute(self, statement):
             time.sleep(0.2)  # 200ms
 
         if status == 'SUCCEEDED':
-            results = self.athena.get_query_results(execution_id)
-            headers = [h['Name'].encode("utf-8") for h in results['ResultSet']['ResultSetMetadata']['ColumnInfo']]
-
-            if self.format in ['CSV', 'CSV_HEADER']:
-                csv_writer = csv.writer(sys.stdout, quoting=csv.QUOTE_ALL)
-                if self.format == 'CSV_HEADER':
-                    csv_writer.writerow(headers)
-                csv_writer.writerows([[text.encode("utf-8") for text in row] for row in self.athena.yield_rows(results, headers)])
-            elif self.format == 'TSV':
-                print(tabulate([row for row in self.athena.yield_rows(results, headers)], tablefmt='tsv'))
-            elif self.format == 'TSV_HEADER':
-                print(tabulate([row for row in self.athena.yield_rows(results, headers)], headers=headers, tablefmt='tsv'))
-            elif self.format == 'VERTICAL':
-                for num, row in enumerate(self.athena.yield_rows(results, headers)):
-                    print('--[RECORD {}]--'.format(num+1))
-                    print(tabulate(zip(*[headers, row]), tablefmt='presto'))
-            else:  # ALIGNED
-                print(tabulate([x for x in self.athena.yield_rows(results, headers)], headers=headers, tablefmt='presto'))
+            output_results(self.athena, self.format, execution_id, sys.stdout, is_shell=False)
 
         if status == 'FAILED':
             print(stats['QueryExecution']['Status']['StateChangeReason'])
 
-try:
-    del cmd.Cmd.do_show  # "show" is an Athena command
-except AttributeError:
-    # "show" was removed from Cmd2 0.8.0
-    pass
-
 
 class AthenaShell(cmd.Cmd, object):
 
-    multilineCommands = ['WITH', 'SELECT', 'ALTER', 'CREATE', 'DESCRIBE', 'DROP', 'MSCK', 'SHOW', 'USE', 'VALUES']
+    multiline_commands = ['WITH', 'SELECT', 'ALTER', 'CREATE', 'DESCRIBE', 'DROP', 'MSCK', 'SHOW', 'USE', 'VALUES', 'with', 'select', 'alter', 'create', 'describe', 'drop', 'msck', 'show', 'use', 'values']
```

Comment:
Why is it necessary to repeat the list of valid commands as both upper- and lowercase?

Reply:
cmd2 seems to be case-sensitive with this feature, i.e. without the lower case commands in the list, then lower case statements aren't multiline

Reply:
This *could* be rewritten as ...
```
    multiline_commands = ['WITH', 'SELECT', 'ALTER', 'CREATE', 'DESCRIBE', 'DROP', 'MSCK', 'SHOW', 'USE', 'VALUES']
    multiline_commands.extend([c.lower() for c in multiline_commands])
```
... to avoid the possibility of missing the fact you need to add the command twice if new commands are added.

Reply:
Yes that's better. Checked in.